.travis.yml

```diff
@@ -1,26 +1,23 @@
+sudo: required
+dist: trusty
 language: cpp
 
 # NOTE: The COMPILER variable is unused. It simply makes the display on
 # travis-ci.org more readable.
 matrix:
     include:
         - compiler: gcc
-          env: COMPILER=g++-4.6     STD=c++0x BUILD_TYPE=Coverage
-        - compiler: gcc
-          env: COMPILER=g++-4.6     STD=c++0x BUILD_TYPE=Debug
-        - compiler: gcc
-          env: COMPILER=g++-4.6     STD=c++0x BUILD_TYPE=Release
+          env: COMPILER=g++         STD=c++11 BUILD_TYPE=Debug
         - compiler: gcc
-          env: COMPILER=g++-4.8     STD=c++11 BUILD_TYPE=Debug
+          env: COMPILER=g++         STD=c++11 BUILD_TYPE=Debug
```

Comment:
This line was lying. "COMPILER=4.8" is ignored entirely. If you look at the output you'll see this was invoking 4.6